js/ui/collection/ui.collection_widget.base.js

```diff
@@ -86,7 +67,7 @@ const CollectionWidget = Widget.inherit({
         };
         return extend(this.callBase(), {
             space: space,
-            enter: enter,
+            enter: this._enterKeyHandler,
```

Comment:
Seems like context is lost here. Why test does not show the mistake?

Reply:
Context is not lost - https://github.com/DevExpress/DevExtreme/blob/1d6c8f5d2ea0ca80cad8db96bcf767262dc3686f/js/ui/widget/ui.widget.js#L330